src/selection.js

```diff
@@ -251,6 +251,16 @@ export default class Selection extends Cursor {
     return content.getTags(this.host, this.range, filterFunc)
   }
 
+  // Get the names of all tags that affect the current selection. Optionally
+  // pass a method to filter the returned elements.
+  //
+  // @param {Function filter(node)} [Optional] Method to filter the DOM i
```

Comment:
`i` -> typo?

Reply:
vim 😉 

Sorry I just messed this one up. Will close and resubmit.